repos/fdbt-site/tests/pages/api/multipleOperators.test.ts

```diff
@@ -24,7 +24,7 @@ describe('multipleOperators', () => {
             nocCode: 'TEST',
         });
         expect(res.writeHead).toBeCalledWith(302, {
-            Location: '/fareType',
+            Location: '/home',
```

Comment:
How cine this is being changed to home?

Reply:
in the multiple operators API - 

`redirectTo(res, globalSettingsEnabled ? '/home' : '/fareType');`

Reply:
Good question - When you log in it now takes you to the home page so you can see the new global setting section, rather than taking you straight to create a new fare product